AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessor.swift

```diff
@@ -533,6 +562,69 @@ extension ItemListProcessor: AuthenticatorKeyCaptureDelegate {
         })
         captureCoordinator.navigate(to: .dismiss(dismissAction))
     }
+
+    /// Display an alert asking the user if they would like to save their choice as their default save option.
+    ///
+    /// After handling their answer to this and saving the option to the `AppSettingsStore`, the key will be
+    /// processed and handled based on what is passed in `sendToBitwarden`.
+    ///
+    /// - Parameters:
+    ///   - key: The key that was captured
+    ///   - sendToBitwarden: `true` if the user previously chose to save the key to the Bitwarden app,
+    ///     `false` if they have chosen to store it locally.
+    ///
+    private func confirmDefaultSaveAlert(key: String, sendToBitwarden: Bool) {
+        let title = sendToBitwarden ?
+            Localizations.setSaveToBitwardenAsYourDefaultSaveOption :
+            Localizations.setSaveLocallyAsYourDefaultSaveOption
+        let option: DefaultSaveOption = sendToBitwarden ? .saveToBitwarden : .saveLocally
```

Comment:
⛏️ I think there's no need to declare it's a `DefaultSaveOption`
```suggestion
        let option = sendToBitwarden ? .saveToBitwarden : .saveLocally
```

Reply:
If I take that out I get a compile error (because the compiler can't infer what to do with the `.saveToBitwarden` and `.saveLocally`)

Reply:
Oh you're right, sorry missed that.